spec/child-apps/happy-unload/happy-unload.app.js

```diff
@@ -17,10 +17,8 @@ export async function unmount() {
 	numUnmountCalls++;
 }
 export function unload() {
-	return new Promise((resolve, reject) => {
-		numUnloadCalls++;
-		setTimeout(resolve, 200);
-	});
+	numUnloadCalls++;
```

Comment:
Getting rid of the setTimeout revealed a lot of race conditions, so I decided to keep it.